### ISSUE TITLE
Support for chinese?

### ISSUE THREAD

**Issue by micxyj** [chakki-works/sumeval#8]
Thank you for making this package, so how can I use Chinese by inheriting BaseLang? Can you give me some detailed steps?

**Comment by policeme** [comment]
@icoxfog417 me too

**Comment by icoxfog417** [comment]
I want to support the Chinese! The spaCy can tokenize Chinese, so we think we can support Chinese like English manner. 

And we need the following material to test behavior.

* Stop word/Stemming word list.
* [regex to replace symbols](https://github.com/chakki-works/sumeval/blob/master/sumeval/metrics/lang/lang_en.py#L9).

@micxyj , @policeme Do you have any resource that would be helpful to implement the above two?

**Comment by policeme** [comment]
I just have vocab stop list, Is the regex, just regex chinese or other word?

**Comment by micxyj** [comment]
- For Chinese stop words, you can refer this link: https://github.com/micxyj/extract_keywords/blob/master/stopwords.txt

- For regex, do you mean that you will replace Chinese punctuation like "，！。", etc?

**Comment by icoxfog417** [comment]
Thank you for offering stop word list!

> For regex, do you mean that you will replace Chinese punctuation like "，！。", etc?

Yes. We have to exclude characters that have to be ignored when calculating ROUGE/BLEU.

**Comment by micxyj** [comment]
```
import re

example = "我：》？喜。，欢！@打~···篮「」球【】"
reg = re.compile(r'[\s+\.\!\/_,$%^*(+\“\’\”\'\"]+|[+——！，。：？、~@#￥%……&*（）【】「」《》·]+')
result = re.sub(reg, '', example)
print(result)
```
Can it works?

**Comment by policeme** [comment]
Now, it support chinese? @icoxfog417

**Comment by icoxfog417** [comment]
I'm now tackling. Please wait a little.

**Comment by policeme** [comment]
ok, thx your working!

**Comment by icoxfog417** [comment]
I implemented Chinese support. And now we need test summaries to test it.
Making test summaries requires careful work. It is needed to adjust the number of tokens. 

* number of tokens of summaries.
* number of tokens of references.
* number of match tokens between summaries and references.

[Here is Japanese sample](https://github.com/chakki-works/sumeval/blob/master/tests/data/rouge/ROUGE-test-ja.json) (Each tokens is splited by space). So it will take a little time.

**Comment by icoxfog417** [comment]
Add test case by referring [Regularizing Output Distribution of Abstractive Chinese Social Media Text Summarization for Improved Semantic Consistency](https://arxiv.org/pdf/1805.04033v1.pdf).

**Comment by icoxfog417** [comment]
Merged #10. Please open the issue if you have some trouble with Chinese!
